spec/derated_construction_spec.rb

```diff
@@ -1,9 +1,4 @@
-begin
-  # Try to load from the tbd gem.
-  require "tbd"
-rescue LoadError
-  require_relative "../lib/tbd"
-end
+require "tbd"
```

Comment:
Spec files run within ```bundle exec ...```, so relying on TBD gem files. A bit circular, but it makes things clearer.